examples/FSharpDev/HttpTests/SimpleHttpTest.fs

```diff
@@ -3,6 +3,7 @@ module FSharpDev.HttpTests.SimpleHttpTest
 open System.Net.Http
 open NBomber
 open NBomber.Contracts
+open NBomber.Contracts.Metrics
```

Comment:
I would keep all this under NBomber.Contracts.Thresholds

Reply:
Yes, this sounds reasonable 👍

Reply:
This should be fixed now.